contracts/MultiManageable.sol

```diff
@@ -6,6 +6,44 @@ import "@poolzfinance/poolz-helper-v2/contracts/Fee/FeeBaseHelper.sol";
 
 /// @title all admin settings
 contract MultiManageable is FeeBaseHelper, Pausable {
+    event MultiTransferredERC20(
+        address token,
+        uint256 userCount,
+        uint256 totalAmount
+    );
+
+    event MultiTransferredETH(uint256 userCount, uint256 totalAmount);
+
+    error ETHTransferFail(address user, uint amount);
+    error ArrayZeroLength();
+    error InvalidTokenAddress();
+    error TotalMismatch(uint amountProvided, uint amountRequired);
+
+    struct MultiSendData {
+        address user;
+        uint amount;
+    }
+
+    modifier notZeroLength(uint256 _length) {
+        if (_length == 0) revert ArrayZeroLength();
+        _;
+    }
+
+    modifier validateToken(address _token) {
+        if (_token == address(0)) revert InvalidTokenAddress();
+        _;
+    }
+
```

Comment:
I am thinking of deleting both of these modifiers.

`ValidateToken` just checks the address should be zeroAddress. If it is any other address which is not ERC20, the function will revert later. Does not make sense to check just for zeroAddress.

Same reasoning for `notZeroLength`, If we remove it the function will just run without dispersing any tokens.